lms/templates/register-form.html

```diff
@@ -234,7 +233,7 @@ <h2 class="sr">${_("Additional Personal Information")}</h2>
     % endif
   </ol>
 </div>
-<%static:optional_include_mako file="register_extra.html" is_theming_enabled="True" />
```

Comment:
Does this file still exist?

Reply:
We don't run this file anymore...this is from pre-logistration

Reply:
and by this file I mean: `lms/templates/register-form.html`